.github/workflows/ci.yml

```diff
@@ -18,6 +18,13 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Checkout local jacodb
```

Comment:
Do not forget to change the local jacodb to a published one (see `settings.gradle.kts` and `Dependencies.kt`) upon the review and remove these checkout jobs on CI before merging.